packages/components-library/src/components/filters/MultiFilter.vue

```diff
@@ -115,13 +125,23 @@ export default {
       type: Array,
       default: () => []
     },
+    satisfyAllValue: {
+      type: Boolean,
+      default: () => false
+    },
     /**
      * The amount of options to show after filtering the checkbox options.
      */
     maxVisibleOptions: {
       type: Number,
       default: () => 10
-    }
+    },
+    showSatisfyAllCheckbox: {
+      type: Boolean,
+      required: false,
+      default: () => true
```

Comment:
Please set the default to false, else it will show up everywhere by default.

Reply:
Fixed in 1c81229